dms/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 {
     "name": "Document Management System",
     "summary": """Document Management System for Odoo""",
-    "version": "16.0.1.8.3",
+    "version": "16.0.1.8.4",
```

Comment:
Don't change manually module version unless you include a migration script. It will be bumped by the bot.